app/views/admin/subscriptions/show.html.erb

```diff
@@ -27,6 +27,43 @@
       <strong><%= Subscription.human_attribute_name(:invoice) %> :</strong>
       <% if @subscription.invoice.attached? %>
         <%= link_to t('defaults.download'), rails_blob_path(@subscription.invoice, disposition: "attachment") %>
+        <br>
+        <br>
+        <%= link_to t('.generate_credit_note'), new_admin_subscription_credit_note_path(@subscription.id), class: "btn btn-outline-success btn-sm" %>
+        <br>
+        <br>
```

Comment:
Don't abuse the br tags here. Use CSS or a bootstrap class to put some margin onto the link

Reply:
It was really ugly...sorry! =S